inc/codeception/overrides.php

```diff
@@ -0,0 +1,87 @@
+<?php
+/**
+ * Bootstrapping code for codeception tests
+ *
+ * This runs both on all Codeception test requests, and acceptance test requests as well.
+ */
+
+// phpcs:disable PSR1.Files.SideEffects
+
+
+// Flush cache now, and register shutdown event to clear the cache, also triggers if an error occurs.
+register_shutdown_function( function () {
+	if ( function_exists( 'wp_cache_flush' ) ) {
+		wp_cache_flush();
+	}
+} );
+
+// If running from the acceptance test thread, ie: in a full WP environment.
+if ( ! function_exists( 'tests_add_filter' ) ) {
+	/**
+	 * Polyfill the tests_add_filter on acceptance tests request threads for consistency.
+	 *
+	 * phpcs:disable HM.Functions.NamespacedFunctions
+	 *
+	 * @return mixed
+	 */
+	function tests_add_filter() {
+		return call_user_func_array( 'add_filter', func_get_args() );
```

Comment:
Fun side note, the spread operator can also be used eg

```suggestion
		return add_filter( …func_get_args() );
```